README.md

```diff
@@ -1,71 +1,46 @@
-## Requirements
-
-**IMPORTANT: Make sure you have the latest version of NEAR Shell and Node**
-
-* `npm` \(Get it [here](https://www.npmjs.com/get-npm)\)
-* `node version 10.x`\(Get it [here](https://nodejs.org/en/download)\)
-* `near-shell`  The NEAR cli tool.
-  * Install with`npm i -g near-shell`
-  * Check out the commands [here](https://github.com/nearprotocol/near-shell)
-  * remember to always update to the latest version
-* Whatever frontend build tools you prefer. We provide React template by default.
-  * By default, `gulp` is used for compiling.  Check it out [here](https://gulpjs.com/). \(You shouldn't need to do any config for gulp specifically\).
-
-## Create New Assembly project
-
-Choose either the React JS or vanilla template
-
-#### React JS app:
-```bash
-npm init near-app path/to/your/new-awesome-app
-```
-with npx:
-
-```bash
-npx create-near-app path/to/your/new-awesome-app
-```
-with yarn:
-```bash
-yarn create near-app path/to/your/new-awesome-app
-```
-
-#### Plain app:
-```bash
-npm init near-app --vanilla path/to/your/new-awesome-app
-```
-with npx:
-
-```bash
-npx create-near-app --vanilla path/to/your/new-awesome-app
-```
-with yarn:
-```bash
-yarn create near-app --vanilla path/to/your/new-awesome-app
-```
-
-## Create New Rust project
-
-Provide vanilla template with smart contract in Rust
-```bash
-npm init near-app --vanilla --rust path/to/your/new-awesome-app
-```
-with npx:
-
-```bash
-npx create-near-app --vanilla --rust path/to/your/new-awesome-app
-```
-with yarn:
-```bash
-yarn create near-app --vanilla --rust path/to/your/new-awesome-app
-```
-
-
-## Develop your own Dapp
-
-#### Simple Tip
-After create project, run ```npm run dev:deploy:contract``` to start and get familiar with features.
-
-#### Next step
-Follow the instructions in the README.md in the project you just created and enjoy the development with NEAR. 
-
-We have [documentation](https://docs.nearprotocol.com) with lots of examples. We have support at [Discord](http://near.chat) and really welcome.
\ No newline at end of file
+create-near-app
+===============
+
+Quickly build apps backed by the [NEAR](https://nearprotocol.com) blockchain
+
+
+Prerequisites
+=============
+
+Make sure you have a [current version of Node.js](https://nodejs.org/en/about/releases/) installed – we have tested most with version `12.x`
```

Comment:
I like this PR a lot. The only thing I'm hesitant about is the verbiage here with version 12. I'd like to have a more solid answer. "Yes, I promise it'll work with 10.x" where this doesn't give me that same feeling. Thoughts?

Reply:
I purposely left it vague because I'm not sure – thanks for asking about this line, @mikedotexe. 

@vgrichina do you know if we can guarantee that `create-near-app` works with a certain version of node?

Reply:
@chadoh looks like we test with node 10.16:
https://github.com/nearprotocol/create-near-app/blob/896b023c4f9acc53c8515e9476447b90f3b82f98/.travis.yml#L13